src/shared/modules/cypher/cypherDuck.js

```diff
@@ -42,6 +42,7 @@ import {
 
 const NAME = 'cypher'
 export const CYPHER_REQUEST = `${NAME}/REQUEST`
+export const ROUTED_CYPHER_REQUEST = `${NAME}/ROUTED_REQUEST`
```

Comment:
Can you rename this to reflect that it's write transactions that are being performed @OskarDamkjaer ?

Reply:
Done!